src/commands/doctor.ts

```diff
@@ -90,8 +90,8 @@ module.exports = {
     // -=-=-=- android -=-=-=-
     const androidPath = process.env.ANDROID_HOME
     const javaPath = which("java")
-    const javaVersionCmd = isWindows ? "java -version" : "java -version 2>&1"
```

Comment:
I'm not entirely sure why this was supposed to be different on Windows, but the command `java -version` resulted in a blank string, while `java -version 2>&1`  resulted in the string we wanted.

Reply:
I didn't dig into git history (it's murky because of previous ignite rewrites) but I would've been suspicious that IO redirection might not have been a thing on Windows. Glad to hear it works and the code can be simpler!